### PR TITLE
Chemical - Fix mortar issues

### PR DESCRIPTION
**Pull request by MiszczuZPolski**
**When merged this pull request will:**
-  Fix wrong inheritance
- Fix "No matching carry mag" that was blocking unloading 8rnd and 5rnd ammo from mortar

### IMPORTANT

- [Development Guidelines](https://ace3.acemod.org/wiki/development/) are read, understood and applied.
- Title of this PR uses our standard template `Component - Add|Fix|Improve|Change|Make|Remove {changes}`.

### CODE REVIEW

**Review by AtrixZockt — APPROVED** [review]
This is what I did but it seems my pr was rigged or something but for me these changes worked when I did them few months ago

**Comment by diaverso** [comment]
This not fix 
https://i.imgur.com/ZuvJSei.jpeg

**Comment by MiszczuZPolski** [comment]
> This not fix https://i.imgur.com/ZuvJSei.jpeg

Yes, i know there are still some issues but i dont have time to search for solutions.

File: addons/chemical/CfgWeapons.hpp [hunk 1]
```diff
@@ -31,18 +31,13 @@ class CfgWeapons {
...
     };
-    class ACE_mortar_82mm;
-    class ACE_mortar_82mm: ACE_mortar_82mm {
-        magazines[] += {
-            "KAT_1Rnd_82mm_Mo_Type0",
-            "KAT_1Rnd_82mm_Mo_TypeCS"
-        };
+
+    class ACE_mortar_82mm: mortar_82mm {
+        magazines[] = {"ACE_1Rnd_82mm_Mo_HE","ACE_1Rnd_82mm_Mo_Smoke","ACE_1Rnd_82mm_Mo_Illum","ACE_1Rnd_82mm_Mo_HE_Guided","ACE_1Rnd_82mm_Mo_HE_LaserGuided", "KAT_1Rnd_82mm_Mo_Type0", "KAT_1Rnd_82mm_Mo_TypeCS"};
```
Comment (LinkIsGrim) [inline addons/chemical/CfgWeapons.hpp:40]:
```suggestion
        magazines[] += {"KAT_1Rnd_82mm_Mo_Type0", "KAT_1Rnd_82mm_Mo_TypeCS"};
```

Comment (MiszczuZPolski) [reply addons/chemical/CfgWeapons.hpp:40]: Leaving as it is because it ends up being only these two

**Comment by nobodyshome** [comment]
Somehow this change has broken/effected the L16 static weapon from [3CB weapons](https://steamcommunity.com/sharedfiles/filedetails/?id=893339590) 

Removed recently updated mods one by one till it worked as intended and it worked after disabling KAT. 

When working, info is described as the "L16 Static Weapon" with "Charge 0" and "81mm HE/Smoke", the charges go up to "Charge 6"

with KAT v2.15.0, it reverts the mortar to just "82mm Mortar" and has "Semi (close)" "Semi (medium)" and "Semi (far)". Believe whatever changes were made regarding this issue may be the cause. Can test and help debug if required.

**Comment by MiszczuZPolski** [comment]
Please contact on discord and create bug thread on it, so its easier to discuss

**Pull request merged**